.circleci/config.yml

```diff
@@ -0,0 +1,70 @@
+version: 2.1
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      - validate-installer-signature
+      - rvm-test-fast-linux
+      - rvm-test-rvm1-linux
+      - rvm-test-long-named-linux
+      - rvm-test-long-truffleruby-linux
+
+jobs:
+  validate-installer-signature:
+    docker:
+        - image: circleci/ruby:2.4 # the primary container, where your job's commands are run
+    steps:
+      - checkout # check out the code in the project directory
+      - run:
+          name: Install gnupg2
+          command: |
+            sudo apt-get install gnupg2
+      - run:
+          name: Update keys
+          command: |
+            gpg2 --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+      - run:
+          name: Verify signature
+          command: |
+            gpg2 --verify binscripts/rvm-installer.asc binscripts/rvm-installer
+
+  default: &default-run-test
+    docker:
+      - image: circleci/ruby:2.4 # the primary container, where your job's commands are run
+    steps:
+      - checkout # check out the code in the project directory
+      - run:
+          name: Install & load RVM
+          command: |
```

Comment:
Putting every command in a new line makes it unnecessary longer and less readable